src/Mesh/Grids.jl

```diff
@@ -173,6 +190,37 @@ function referencepoints(::DiscontinuousSpectralElementGrid{T, dim, N}
   ξ
 end
 
+"""
+    minnodedistance(::DiscontinuousSpectralElementGrid,
```

Comment:
Something like `min_node_distance` would be easier to read.

Reply:
Changed, thanks!